### ISSUE TITLE
tsai on Sagemaker llvmlite installation issue

### ISSUE THREAD

**Issue by navinshahse** [timeseriesAI/tsai#305]
Ran into this problem while installing tsai on Sagemaker Notebook instance , Kernel : conda_pytorch_latest_36

![image](https://user-images.githubusercontent.com/68047787/145357805-7a713351-192d-45ce-9f55-24156c580b09.png)

**Comment by oguiza** [comment]
Hi @navinshahse,
Do you know what version of Pytorch is installed? tsai has a requirement torch>=1.7.0,<1.10. If Pytorch 1.10 is installed it will try to downgrade, and it may not be possible. I don't know if you can choose an environment with Pytorch 1.9. 
There's currently an issue with Pytorch 1.10 that needs to be fixed. As soon as this is done, the constraint will be removed.

**Comment by oguiza** [comment]
Hi, 
I've already upgraded tsai to work with Pytorch 1.10. However, when testing it on SageMaker I've learned the issue is related to two of tsai's dependencies: sktime and numba. Both require a llvmlite uninstall during the installation which does not seem possible in SageMaker.
The only solution to this problem seems to be dropping both sklearn and numba from tsai's dependencies. That requires a more in-depth review to assess if it's possible or not.
